src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Views/Dashboard/InModerationApplications.cshtml

```diff
@@ -22,7 +22,8 @@
         <div class="govuk-grid-column-full">
             <div class="govuk-tabs">
 
-                @{ await Html.RenderPartialAsync("DashboardNavigation.cshtml"); }
+                @{
+                    await Html.RenderPartialAsync("DashboardNavigation.cshtml"); }
```

Comment:
Could we use the .ASP Core: <partial> tag here?  @Html.RenderPartialAsync has been deprecated